api-challenge/README.md

```diff
@@ -0,0 +1,83 @@
+# Safe Harbor De-Identification
+
+## Introduction
+
+In the United States, the privacy of patient data is governed by the Health
+Insurance Portability and Accountability Act, or as its commonly known, HIPAA.
+One regulation falling under HIPAA is the Privacy Rule, which among other
+things, governs what health data can be shared and under what circumstances.
+Researchers in health fields often seek out "Limited Data Sets", which have been
+stripped of 18 types of data, including biometrics such as fingerprints, birth
+dates, telephone numbers, social security numbers, etc.  This process is known
+as 'Safe Harbor' de-identification.
+
+For this assessment, we're going to write a real-time de-identification engine,
+which is a buzzwordy way of saying we're going to build a simple API which takes
+in a single patient record as JSON, and returns a de-identified version.  We're
+not going to try to implement all 18, but rather the following rules.
+
+## Problem Statement
+
+Write an API that exposes an endpoint for the de-identification of patient
+records. Specifically, we're looking to for the following:
+
+* Birthdates should be converted to the patient's age.  If someone is over the age
+  of 89, they should be lumped into an 90+ category
+
+* ZIP Codes should be stripped to the first three digits _except_ where fewer
+  than 20,000 reside in the combination of all ZIP Codes with those three
```

Comment:
Add "people" after 20,000? Also I don't think you need to capitalize "Codes".
```suggestion
  than 20,000 people reside in the combination of all ZIP codes with those three
```

In addition, the word "combination" is throwing me off here. We're talking about the geographic area covered by all ZIP codes with the same three-digit prefix, correct?